static/js/pages/PodcastFrontpage.js

```diff
@@ -29,6 +29,14 @@ export default function PodcastFrontpage() {
 
   useLearningResourcePermalink()
 
+  // prevent spacebar from scrolling the page
+  window.addEventListener("keydown", e => {
+    if (e.keyCode === 32) {
```

Comment:
We can use `e.key` here instead. React will ensure that the attribute exists: https://reactjs.org/docs/events.html#keyboard-events